database/seeds/eggs/source-engine/egg-garrys-mod.json

```diff
@@ -49,6 +49,51 @@
             "user_viewable": 0,
             "user_editable": 0,
             "rules": "required|string|max:20"
+        },
+        {
+            "name": "Workshop ID",
+            "description": "The ID of your workshop collection (the numbers at the end of the URL)",
+            "env_variable": "WORKSHOP_ID",
+            "default_value": "",
+            "user_viewable": 1,
+            "user_editable": 1,
+            "rules": "nullable|integer"
+        },
+        {
+            "name": "Workshop API Key",
+            "description": "Steam dev API key (https:\/\/steamcommunity.com\/dev\/apikey)",
+            "env_variable": "WORKSHOP_API",
+            "default_value": "",
+            "user_viewable": 1,
+            "user_editable": 1,
+            "rules": "nullable"
+        },
+        {
+            "name": "Gamemode",
+            "description": "The gamemode of your server.",
+            "env_variable": "GM_NAME",
+            "default_value": "sandbox",
+            "user_viewable": 1,
+            "user_editable": 1,
+            "rules": "required|string"
+        },
+        {
+            "name": "Max Players",
+            "description": "The maximum amount of players allowed on your game server.",
+            "env_variable": "PLYS",
```

Comment:
This is a terrible environment key name, theres no reason to shorten it like that.